modulus/metrics/diffusion/loss.py

```diff
@@ -749,3 +759,110 @@ def __call__(self, net, images, labels, augment_pipe=None):
         loss = (e - output).square()
 
         return loss
+
+
+class RegressionLossCE:
+    """
+    Regression loss function for GEFS-HRRR model with probabilities, modified from RegressionLoss.
```

Comment:
Would be helpful if you can explain how this loss is different from `RegressionLoss`

Reply:
Thanks. I have added the following comments to the function: A regression loss function for the GEFS-HRRR model with probability channels, adapted from RegressionLoss. In this version, probability channels are evaluated using CrossEntropyLoss instead of MSELoss.